subsys/bluetooth/services/fast_pair/fp_crypto/fp_crypto_oberon.c

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "fp_crypto.h"
+
+#include <ocrypto_hmac_sha256.h>
+#include <ocrypto_sha256.h>
+#include <ocrypto_aes_ecb.h>
+#include <ocrypto_ecdh_p256.h>
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_DECLARE(fp_crypto, CONFIG_FP_CRYPTO_LOG_LEVEL);
+
+int fp_crypto_sha256(uint8_t *out, const uint8_t *in, size_t data_len)
+{
+	ocrypto_sha256(out, in, data_len);
+
+	return 0;
+}
+
+int fp_crypto_hmac_sha256(uint8_t *out,
+			  const uint8_t *in,
+			  size_t data_len,
+			  const uint8_t *hmac_key,
+			  size_t hmac_key_len)
+{
+	ocrypto_hmac_sha256(out, hmac_key, hmac_key_len, in, data_len);
```

Comment:
Does ocrypto function  set returns any value? Maybe those function here could be declared as void?

Reply:
It returns void as all the ocrypto_ functions which are called with the ret/err assignment

We cannot change the fp_crypto_hmac_sha256 API since other crypto backends (MbedTLS, Tinycrypt) return regular error codes and it would ruin the abstraction